docs/book/helpers/head-title.md

```diff
@@ -1,3 +1,8 @@
+<!-- markdownlint-configure-file {
+    "no-trailing-spaces": false,
```

Comment:
The unwanted trailing space should be removed, so there is no need to disable this rule.